src/components/details.js

```diff
@@ -1,6 +1,7 @@
-import { useParams } from 'react-router-dom';
+import { useParams, NavLink } from 'react-router-dom';
 import { useSelector, useDispatch } from 'react-redux';
 import { useEffect } from 'react';
+import { FiArrowLeft } from 'react-icons/fi';
 import { fetchCountries } from '../redux/countries/countrySlice';
 
 const Details = () => {
```

Comment:
- [ ] I recommend improving your styling to avoid leaving a hollow page. The lower part of your app contains no information, instead of leaving it blank (white), kindly add your (pinkish) background color.

  ![image](https://user-images.githubusercontent.com/36542986/227204006-e0dc72bf-f19a-443a-b6de-3b4f9dba510a.png)